app/src/main/java/edu/grinnell/appdev/grinnelldirectory/activities/SearchPagerActivity.java

```diff
@@ -103,6 +132,22 @@ private void setAnimation() {
         }
     }
 
+    @OnClick(R.id.retry_button)
+    void testConnection() {
```

Comment:
Can this be `private`?

Reply:
No, because ButterKnife requires it to be non-private